practiceapp/src/test/java/com/practiceapp/practiceapp/service/PostServiceTest.java

```diff
@@ -51,4 +51,13 @@ void savePost() {
 
     }
 
+    @Test
+    void updatePost() {
+
+        when(postRepository.save(any(PostEntity.class))).thenReturn(postEntity);
+        int response = postService.updatePost(postEntity);
+        assertEquals(response,1);
+
```

Comment:
Since updatePost function always returns 1, this test is always going to be passed. 
Please consider Zeynep's comment